.gitignore

```diff
@@ -11,6 +11,7 @@ hookdocs
 node_modules/
 tests/images/test-*
 vendor/
+vendor_prefixed/
```

Comment:
Folder name suggestions are welcome. 🙂 

Maybe `vendor_scoped`? 🤔

Reply:
Having the vendor in the name is a bit confusing, so I've updated the folder and namespace to `third-party` / `Sensei\ThirdParty`.